spec.md

```diff
@@ -4215,7 +4226,8 @@ decl_layer =
 
 decl_type_alias = "type", id, "=", type ;
 
-port = ( "input" | "output" ) , id , ":" , (type | type_property) , [ info ] ;
+port = ( "input" | "output" ) , id , ":" , (type | type_property) ,
+       [ info ] ;
 type_param = int | string_dq | string_sq ;
```

Comment:
Please revert this and all other changes to the grammar.

These do not fall off the page in the PDF. But I laid the grammar out manually so that the newlines in the grammar section are suggestive of where the `newline` tokens appear.

If you think I missed something regarding these changes, please let me know.

Reply:
I reverted most changes as per your request, with the exception of the addition of a missing literal comma in the definition of reg.

The very end of `primop_1expr2int`'s definition does fall entirely off the page, however.

To be clear, I originally formatted this section so that everything fit in the standard width of text, as indicated by the horizontal rule under the pages' headers (and many definitions did indeed not quite fit). However, I can fully understand why making the position of newline tokens suggestive takes precedence over this concern.

Reply:
Thank you!